bofire/domain/constraints.py

```diff
@@ -230,9 +217,14 @@ def from_greater_equal(
             rhs=-1.0 * rhs,
         )
 
+    # TODO: from_smaller_equal should take the object as input
     @classmethod
     def from_smaller_equal(
-        cls, features: List[str], coefficients: List[float], rhs: float
+        cls,
+        features: List[str],
+        coefficients: List[float],
+        rhs: float,
+        **kwargs,
```

Comment:
Same question here regarding the kwargs ;)

Reply:
see above :-P